_layouts/recipes-1.2.html

```diff
@@ -37,7 +37,7 @@ <h4 class="panel-title">
     <p>Fluo Recipes {{ page.version }} documentation &nbsp;&gt;&gt;&nbsp; {{ page.category | capitalize | replace: "-", " " }} &nbsp;&gt;&gt;&nbsp; {{ page.title }}</p>
     {% endif %}
 
-    <div class="alert alert-danger" style="margin-bottom: 0px;" role="alert">This documentation is for a future release of Fluo! <a href="{{ site.baseurl }}/docs/fluo/{{ site.latest_fluo_release }}/">View documentation for the latest release</a>.</div>
+    <!--div class="alert alert-danger" style="margin-bottom: 0px;" role="alert">This documentation is for a future release of Fluo Recipes! <a href="{{ site.baseurl }}/docs/fluo-recipes/{{ site.latest_recipes_minor }}/">View documentation for the latest release</a>.</div>-->
```

Comment:
Why not remove?

Reply:
This div is needed when `recipes-1.3.html` is created (by copying this file) so I didn't want to remove it completely.

Reply:
ok